packages/block/src/header.ts

```diff
@@ -243,6 +186,46 @@ export class BlockHeader {
       )
     }
 
+    const defaults = {
+      parentHash: zeros(32),
+      uncleHash: KECCAK256_RLP_ARRAY,
+      coinbase: Address.zero(),
+      stateRoot: zeros(32),
+      transactionsTrie: KECCAK256_RLP,
+      receiptTrie: KECCAK256_RLP,
+      logsBloom: zeros(256),
+      difficulty: BigInt(0),
+      number: BigInt(0),
+      gasLimit: DEFAULT_GAS_LIMIT,
+      gasUsed: BigInt(0),
+      timestamp: BigInt(0),
+      extraData: Buffer.from([]),
+      mixHash: zeros(32),
+      nonce: zeros(8),
+      baseFeePerGas: undefined,
+    }
```

Comment:
Ah, that's nice. We might want to do this in other libraries as well, that gives a really good overview! 😄